### PR TITLE
Adding php-amqp

### PR DESCRIPTION
**Pull request by erikaheidi**
Adding the `php-amqp` extension.

#### For new package PRs only
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates

### CODE REVIEW

File: php-amqp.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    runs: |
+      set -x
+      INSTALL_ROOT="${{targets.destdir}}" DESTDIR="${{targets.destdir}}" make install
+
+update:
+  enabled: true
+  github:
+    identifier: php-amqp/php-amqp
+    strip-prefix: v
+    tag-filter: v
```
Comment (rawlingsj) [inline php-amqp.yaml:50]:
```suggestion
    tag-filter: v
    use-tag: true
```
think adding this will fix the CI check

**Comment by erikaheidi** [comment]
This package seems to handle releases differently, they use tags but don't make releases? Not quite sure how to handle the update here

File: php-amqp.yaml (hunk 1 above)
Comment (erikaheidi) [reply php-amqp.yaml:50]: cool, let me try that!

**Review by rawlingsj — DISMISSED** [review]

**Comment by erikaheidi** [comment]
Awesome, thanks @rawlingsj for the quick 👍🏼 :)

**Comment by rawlingsj** [comment]
> Awesome, thanks @rawlingsj for the quick 👍🏼 :)

Sure thing!  Looks like it may be removed from the merge queue because of a conflict of another change that just went in.  I tried to time them to go in together.  I'll sort out the conflict though and make sure it get's merged.

**Review by rawlingsj — APPROVED** [review]

**Pull request merged**